sound/soc/sof/intel/bdw.c

```diff
@@ -519,6 +519,10 @@ static int bdw_fw_ready(struct snd_sof_dev *sdev, u32 msg_id)
 		 " Firmware info: version %d:%d-%s build %d on %s:%s\n",
 		 v->major, v->minor, v->tag, v->build, v->date, v->time);
 
+	/* only copy the fw_version into debugfs at first boot */
+	if (sdev->first_boot)
```

Comment:
need to check that this not NULL in all files, this element may not be created successfully.

Reply:
`bool first_boot;`
first_boot is a bool value and should be init with 0.
So I did not think it needs to be check